azext_iot/common/shared.py

```diff
@@ -306,3 +306,16 @@ class SHAHashVersions(Enum):
     """
     SHA1 = 1
     SHA256 = 256
+
+
+class BulkDeviceImportMode(Enum):
```

Comment:
Reminder to remove the enum if it will not be used by the time the PR is ready for merge.